core/domain/fs_domain.py

```diff
@@ -286,7 +420,7 @@ def _check_filepath(self, filepath):
         if not normalized_path.startswith(base_dir):
             raise IOError('Invalid filepath: %s' % filepath)
 
-    def isfile(self, filepath):
+    def isfile(self, filepath: str) -> bool | Any:
```

Comment:
Why `Any` is returned?

Reply:
![image](https://user-images.githubusercontent.com/37413698/158050790-ad555f38-95a6-43ad-9905-86c70f85b83c.png)
It is because of this